### PR TITLE
pdftools: initial add

### PR DESCRIPTION
**Pull request by johannesring**

### CODE REVIEW

File: recipes/pdftools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{% set version = "0.37" %}
+{% set sha256 = "634cb50bcd276dda5ab39e3602acfb0370694b61b70eb357201f2bd40240ab4e" %}
+
+package:
+  name: pdftools
```
Comment (jakirkham) [inline recipes/pdftools/meta.yaml]: So this overlaps with a PyPI package also call `pdftools`, but they don't appear to be the same AFAICT. What would like to do about this, @conda-forge/core?

Comment (jakirkham) [inline recipes/pdftools/meta.yaml]: Any thoughts on this @conda-forge/core? Would really appreciate feedback from others on how to handle this. 😄

Comment (scopatz) [inline recipes/pdftools/meta.yaml]: I am not sure that we can fully avoid name conflicts.  PyPI takes the stance that the name is claimed by whoever got their first.  Out of shear practicality, maybe that is the best we can do.

Comment (johannesring) [inline recipes/pdftools/meta.yaml]: This is the package called `pdftools` in Ubuntu: http://packages.ubuntu.com/xenial/python-pdftools.

Comment (patricksnape) [inline recipes/pdftools/meta.yaml]: Yuck 😞 I don't suppose the conda people have some scheme for namespacing yet 😆? In this case, perhaps we *do* prefix this with *python-*?

Comment (scopatz) [inline recipes/pdftools/meta.yaml]: @patricksnape I think that part of the problem here is that there are *two* python packages which have the same name.  The python namespace doesn't help to disambiguate.

Comment (jakirkham) [inline recipes/pdftools/meta.yaml]: Exactly. Hence the question.

Comment (patricksnape) [inline recipes/pdftools/meta.yaml]: Ignore me, apparently I didn't aid there at all. I vote to stick with pypi - if someone has a non pypi package that they are maintaining then they are sort of 'outside' the Python packaging ecosystem. In this case I feel like the best we can do is raise this upstream and wait?

Comment (jakirkham) [inline recipes/pdftools/meta.yaml]: Yep, I agree. It's a mess, but that seems like all we can do.

@pelson and @ocefpaf , any other thoughts on what to do here?

**Pull request closed without merging**